lib/teamcity/firebird/firebird_installation.txt

```diff
@@ -8,7 +8,7 @@ Installation steps for Firebird for NH TeamCity:
 5.  Go into Firebird folder (c:\program files\firebird\) and create a folder named Data;
 6.  Go in Firebird installation directory and open databases.conf;
 7.  Add in "Live Databases" section:
-nhibernate = C:\Program Files\Firebird\Data\nhibernate.fdb
+nhibernate = D:\SqlData\Firebird\nhibernate.fdb
```

Comment:
Local to the database server? Would be on `C:`, which is not local to the host. Local to the NHibernate test client? The path seems not frozen, only embedded mode can do that, where the client is the server too. I think Firebird mainstream usage is as an actual server, not as embedded mode, so I would rather test it as an actual server. And as showcase by Nathan branch, Firebird v3 embedded mode seems to need a new dialect.

When using a database server, does it really make sens to locate the database relatively to the client? That should be the server choice, not the client. Currently this is done with a configured alias in FireBird_3_0\Databases.conf file, which decides where to put the file for the alias `nhibernate`.